modules/apps/portal-search/portal-search-test-util/src/main/java/com/liferay/portal/search/test/util/DocumentsAssert.java

```diff
@@ -93,7 +93,7 @@ public static void assertValuesIgnoreRelevance(
 
 	public static void assertValuesIgnoreRelevance(
 		String message, List<Document> documentsList, String fieldName,
-		Stream<?> expectedValues) {
+		Collection<?> expectedValues) {
```

Comment:
@julschong Please try to make these methods consistent, most of the usages are passing `List<String>` and it seems only 1 usage is passing `List<Long>`. Just make the `expectedValues` `List<String>` so you can simplify more of `_sort()`.